src/Util/OWSError.m

```diff
@@ -35,7 +37,13 @@
 NSError *OWSErrorMakeAssertionError()
 {
     return OWSErrorWithCodeDescription(OWSErrorCodeFailedToSendOutgoingMessage,
-        NSLocalizedString(@"ERROR_DESCRIPTION_UNKNOWN_ERROR", @"Worst case generic error message"));
+                                       NSLocalizedString(@"ERROR_DESCRIPTION_UNKNOWN_ERROR", @"Worst case generic error message"));
+}
+
+NSError *OWSErrorMakeWebRTCMissingDataChannelError()
+{
+    return OWSErrorWithCodeDescription(OWSErrorCodeWebRTCMissingDataChannel,
+                                       NSLocalizedString(@"ERROR_DESCRIPTION_WEBRTC_MISSING_DATA_CHANNEL", @"Missing data channel while trying to sending message"));
```

Comment:
Is there any point in localizing an error like this? I don't think it would mean anything to our users or our translators.